2200072/민석이의과제체크하기.py

```diff
@@ -0,0 +1,9 @@
+
+t = int(input())
+for i in range(t):
+  n,_ = map(int, input().split())
+  numbers = {num for num in range(1,n+1)}
```

Comment:
완벽합니다! 💯 
```suggestion
  numbers = set(range(1,n+1))
```
위와 같이 작성하는 방법도 있습니다!